components/x-gift-article/package.json

```diff
@@ -23,10 +23,10 @@
   },
   "devDependencies": {
     "@financial-times/o-buttons": "^7.2.2",
-    "@financial-times/o-forms": "^9.2.3",
+    "@financial-times/o-forms": "^9.11.0",
     "@financial-times/o-labels": "^6.2.2",
     "@financial-times/o-loading": "^5.2.1",
-    "@financial-times/o-message": "^5.2.1",
+    "@financial-times/o-message": "5.4.2",
```

Comment:
**Question**:Any reason to remove the `^`?

Reply:
nope, it must've automagically been removed by my IDE